monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -1007,9 +1009,11 @@ sealed abstract class Task[+A] extends Serializable {
     * will be `maxRetries + 1`.
     */
   final def onErrorRestart(maxRetries: Long): Task[A] =
-    this.onErrorHandleWith(ex =>
-      if (maxRetries > 0) this.onErrorRestart(maxRetries-1)
-      else raiseError(ex))
+    this.onErrorHandleWith(
+      ex =>
```

Comment:
Meh.

Reply:
Unfortunately you cannot configure it to always leave the function parameter after the `(`. And it moves stuff to the next line because the line was too long. But this particular case can be solved by using `{` instead of `(`.

Reply:
I've converged a bit more towards the existing style, but this cannot be solved without using the `{ }` lambda syntax.